e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/RunsMenuAO.java

```diff
@@ -354,6 +355,19 @@ public RunsMenuAO pause(final String runId, final String pipelineName) {
         return this;
     }
 
+    public RunsMenuAO ensurePauseButtonDisabled(String runID) {
+        context().find(byId(format("run-%s-pause-button", runID)))
+                .shouldHave(Condition.cssClass("cp-disabled"));
+        return this;
+    }
+
+    public RunsMenuAO checkPauseButtonTooltip(String runID, String message) {
+        hover(byId(format("run-%s-pause-button", runID)));
+        $(PipelineSelectors.visible(byClassName("ant-popover-inner-content")))
+                .shouldHave(Condition.text(message));
```

Comment:
On-demand static import for Condition already exist

Reply:
Done